codegens/csharp-restsharp/lib/restsharp.js

```diff
@@ -158,6 +225,14 @@ self = module.exports = {
         type: 'boolean',
         default: false,
         description: 'Remove white space and additional lines that may affect the server\'s response'
+      },
+      {
+        name: 'Set communication type',
+        id: 'asyncType',
+        type: 'enum',
+        availableOptions: ['async', 'sync'],
+        default: 'async',
+        description: 'Set if the requests will be asynchronous or synchronous'
       }
```

Comment:
{
        name: 'Execution type',
        id: 'execType',
        type: 'enum',
        availableOptions: ['async', 'sync'],
        default: 'async',
        description: 'Choose if the requests will be executed asynchronously or synchronously'
      }

Reply:
@alexeyzimarev Thank you.
@LuisTejedaS Can we remove this option altogether?

Reply:
Hi @umeshp7 i removed the option and the code regarding sync, we are only using async

Reply:
👍 Sync calls are just extensions using the AsyncHelper wrapper. It's a workaround, and people should only be using async calls anyway.